share/instruments/instruments.xml

```diff
@@ -7768,19 +7763,15 @@
                         <stem>1</stem>
                   </Drum>
                   <Articulation>
-                        <gateTime>100</gateTime>
                         <velocity>100</velocity>
+                        <gateTime>100</gateTime>
                   </Articulation>
                   <Articulation name="sforzato">
                         <velocity>130</velocity>
                   </Articulation>
                   <Articulation name="marcato">
                         <velocity>500</velocity>
                   </Articulation>
-                  <channel>
-                        <controller ctrl="0" value="1"/>
-                        <program value="96"/> <!--Marching Tenor-->
-                  </channel>
```

Comment:
same

Reply:
Should have been `<Channel>` not `<channel>` in the old instruments.xml.

Reply:
Fixed and moved a few lines above.